src/main/java/Storage.java

```diff
@@ -57,10 +57,10 @@ public void writeToFile(String fileName, String data) {
 
     public void doTask(int idx) {
```

Comment:
As per the `A-MoreOOP` and `A-Packages` levels of Week 3, you might want to move your methods like `doTask` and `deleteTask` out of your `Storage.java` class, and instead into a separate `TaskList.java` class - which will probably save you from having too many methods constantly touching your data file with `FileWriter`!